plugin.cpp

```diff
@@ -27,72 +28,77 @@ static const char * def_cfg = QUOTE({
 	"email_to" : {
 		"description" : "The address to send the alert to",
 		"type" : "string",
-		"default" : "alert.subscriber@dianomic.com",
+		"default" : "Notification alert subscriber <alert.subscriber@dianomic.com>",
 		"order" : "1",
-		"displayName" : "To address"
+		"displayName" : "To"
 		},
-	"email_to_name" : {
-		"description" : "The name to send the alert to",
+	"email_cc" : {
+		"description" : "The address to send the alert to Cc",
 		"type" : "string",
-		"default" : "Notification alert subscriber",
+		"default" : "Notification alert subscriber <alert.subscriber@dianomic.com>",
 		"order" : "2",
-		"displayName" : "To "
+		"displayName" : "CC"
 		},
-	"subject" : {
-		"description" : "The email subject",
+	"email_bcc" : {
+		"description" : "The address to send the alert to Bcc",
 		"type" : "string",
-		"displayName" : "Subject",
+		"default" : "Notification alert subscriber <alert.subscriber@dianomic.com>",
 		"order" : "3",
-		"default" : "Fledge alert notification"
+		"displayName" : "BCC"
 		},
 	"email_from" : {
 		"description" : "The address the email will come from",
 		"type" : "string",
 		"displayName" : "From address",
-		"default" : "dianomic.alerts@gmail.com",
+		"default" : "Notification Alert <dianomic.alerts@gmail.com>",
 		"order" : "4"
 		},
-	"email_from_name" : {
-		"description" : "The name used to send the alert email",
+	"subject" : {
+		"description" : "The email subject. Macro $NOTIFICATION_INSTANCE_NAME$ can be used to provide information about notification instance name",
 		"type" : "string",
-		"default" : "Notification alert", 
-		"displayName" : "From name",
-		"order" : "5"
+		"displayName" : "Subject",
+		"order" : "5",
+		"default" : "Fledge alert notification triggerd by $NOTIFICATION_INSTANCE_NAME$ "
 		},
 	"server" : {
 		"description" : "The SMTP server name/address",
 		"type" : "string",
 		"displayName" : "SMTP Server",
 		"order" : "6",
-		"default" : "smtp.gmail.com"
+		"default" : "smtp.gmail.com",
+		"group" : "SMTP Details"
```

Comment:
In the Jira you asked about name change of the group and this does not match the answer I gave you. "SMTP Details" is not a good name for a group.

Reply:
Yeah, I see the Jira description later. Mail Server is a better name, and Authentication should be merged into this. As we don't have group orders but sorted by same name.